cmd/spire-server/cli/agent/agent_test.go

```diff
@@ -266,7 +266,7 @@ func TestList(t *testing.T) {
 			expectedReturnCode:   0,
```

Comment:
can you add a test case where an agent that is re_attestable is returned?

Reply:
0db980d